app/src/main/java/com/android/shelfLife/ui/newoverview/AddFoodItem.kt

```diff
@@ -27,18 +34,15 @@ import kotlinx.coroutines.launch
  * @param userRepository The user model.
  * @param paddingValues The padding values to be applied to the screen.
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun AddFoodItemScreen(
     navigationActions: NavigationActions,
-    foodItemRepository: FoodItemRepository,
-    userRepository: UserRepository,
-    paddingValues: PaddingValues = PaddingValues(16.dp)
+    paddingValues: PaddingValues = PaddingValues(16.dp),
```

Comment:
With the new hilt architecture now instead of a FoodItemViewModel we are using hilt, just make sure that it has the same functionality